quickstart/speech-translation/cpp-windows/helloworld/helloworld.cpp

```diff
@@ -0,0 +1,72 @@
+#include "pch.h"
+#include <iostream>
```

Comment:
add copyright headers to all your source files

Reply:
Is this only true for C++ or all languages? Can I use what's in the Recognition samples?